decidim_app-design/Gemfile.lock

```diff
@@ -309,7 +309,7 @@ GEM
     browser (2.7.1)
     builder (3.2.4)
     byebug (11.1.3)
-    capybara (3.35.1)
+    capybara (3.35.0)
```

Comment:
There seems to be some Gems in the design app that are downgraded:
- capybara `3.35.1` -> `3.35.0`
- chef-utils `16.9.32` -> `16.9.29`
- puma `5.0.2` -> `5.0.0`
- terminal-table `3.0.0` -> `2.0.0`

Although the design app will not go live anywhere, if there is no actual reason for the downgrades, let's keep the unrelated gems in the versions they were in. To me it seems accidental.